app/views/books/show_list.html.erb

```diff
@@ -29,4 +29,14 @@
     </tr>
     <% end %>
   </tbody>
-</table>
\ No newline at end of file
+</table>
+
+<% if @multi_loan_available %>
+  <%= form_tag(loan_multi_path, method: "post") do %>
```

Comment:
This form needs a headline or, preferably, to be invisible until a "new loan" button is clicked.